contrib/python-hypothesis/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "python-hypothesis"
+pkgver = "6.92.2"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-attrs",
+    "python-pexpect",
+    "python-pytest",
+    "python-pytest-xdist",
+    "python-sortedcontainers",
+]
+depends = ["python-attrs", "python-sortedcontainers"]
+pkgdesc = "Easy to use library for property-based testing"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MPL-2.0"
+url = "https://hypothesis.works/index.html"
+source = f"https://github.com/HypothesisWorks/hypothesis/archive/refs/tags/hypothesis-python-{pkgver}.zip"
+sha256 = "899688a4f7dcca9a6d32ce486a7765db9eef805ee4d84d3333fa6e06276e67b8"
+
+
+def init_build(self):
+    self.make_build_target = "hypothesis-python"
```

Comment:
this can be top level, doesn't have to be in init_build

Reply:
looking at it, you might want to merely set `build_wrksrc = "hypothesis-python"` though, since everything is in that folder and the repo toplevel is not used